composer.json

```diff
@@ -34,7 +35,7 @@
         "sonata-project/user-bundle": "For user timezone detection"
     },
     "conflict": {
-        "sonata-project/user-bundle": "<2.0 || >=4.0"
+        "sonata-project/user-bundle": "<2.0 || >=5.0"
```

Comment:
And also, I think the root of the issue here is that we are testing 2 development versions together. I think we should avoid that kind of thing.